src/Pages/AboutUsManager/AboutUsManagerModal.js

```diff
@@ -0,0 +1,312 @@
+    [
+      {
+        addon: 'First Name*',
+        type: 'text',
+        defaultValue: props.name,
+        placeholder: 'e.g. John D.',
+        maxlength: '24',
+        handleChange: e => setName(e.target.value)
+      },
+      {
+        addon: 'Email*',
+        type: 'email',
+        defaultValue: props.email,
+        handleChange: e => setEmail(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'LinkedIn Profile',
+        type: 'url',
+        placeholder:'Enter URL Here',
+        defaultValue: props.linkedin,
+        handleChange: e => setLinkedin(e.target.value)
+      }
+    ],
+    [
+      {
+        addon: 'Profile Picture*',
+        type: 'url',
+        placeholder: 'Enter URL Here',
+        defaultValue: props.pictureUrl,
+        handleChange: e => setPictureUrl(e.target.value)
+      }
+    ]
+  ];
+
+  async function handleSubmission() {
+    const officerFields = {
+      name,
+      email,
+      linkedin,
+      team,
+      position,
+      quote,
+      pictureUrl
+    };
+    await props.handleSubmit({...officerFields});
+    window.location.reload();
+    props.toggle();
+  }
+
+  function requiredFieldsFilledIn() {
+    if (team === 'executive' && props.modalState === officerModalState.EDIT) {
+      return (
+        name !== '' &&
+        email !== '' &&
+        team !== '' &&
+        pictureUrl !== '' &&
+        linkedin !== '' &&
+        quote !== '' &&
+        position !== ''
+      );
+    } else if (team === 'executive' &&
+    props.modalState === officerModalState.SUBMIT) {
+      return (
+        name !== undefined &&
+        email !== undefined &&
+        team !== undefined &&
+        pictureUrl !== undefined &&
+        linkedin !== undefined &&
+        quote !== undefined&&
+        position !== undefined
+      );
+    } else if (props.modalState === officerModalState.EDIT){
+      return (
+        name !== '' &&
+            email !== '' &&
+            team !== '' &&
+            pictureUrl !== ''
+      );
+    } else if (props.modalState === officerModalState.SUBMIT){
+      return (
+        name !== undefined &&
+            email !== undefined &&
+            team !== undefined &&
+            pictureUrl !== undefined
+      );
+    }
+    return false;
+  }
+
+  return (
+    <div>
+      <ConfirmationModal {...confirmModalProps} />
+      <Modal isOpen={modal} size='lg' toggle={toggle}>
+        <ModalHeader>
+          {modalState === officerModalState.SUBMIT ? 'Create New ' : 'Edit '}
+          Officer
+        </ModalHeader>
+        <ModalBody>
+          <span>
+            <span color='red'>*</span>= This is a required field
+          </span>
+          <Row className='container'>
+            <Label>Team*</Label>
+            <Input
+              type='select'
+              defaultValue={props.team}
+              onChange={e => setTeam(e.target.value)}
+              children= {
+                <>
+                  <option/>
```

Comment:
any reason for this empty option?

Reply:
I thought it would help indicate that the field needs to be filled in, because otherwise the first option would be the default. But, I can remove it!